data-sdk/src/main/java/com/logicmonitor/sdk/data/ApiClientUserAgent.java

```diff
@@ -15,6 +15,10 @@ public class ApiClientUserAgent extends ApiClient {
   Setup setup = new Setup();
 
   public ApiClientUserAgent() {
+    String applicationName = "";
+    if (System.getenv("APPLICATION_NAME") != null) {
+      applicationName = "/" + System.getenv("APPLICATION_NAME");
```

Comment:
Not secure.. We can't open ended Env size .. it needs to be bounded to a max limit .. say 32 bytes. It can create SHELL related vulnerabilities